### DISCUSSION TITLE
Support Forgejo, Gitea, Gitlab, Radicle

### DISCUSSION THREAD

**Discussion by trymeouteh** [nektos/act#2423] [Ideas]
Support other git services such as Forgejo, Gitea, Github and Radicle.

For example, the directory on Forgejo is `.forgejo/workflows`
https://forgejo.org/docs/latest/user/actions/

**Comment by ChristopherHX** [comment]
> Forgejo, Gitea

Those are using forks of nekos/act, while Forgejo act is fork of fork.

e.g. gitea use the act_runner binary to run locally it has a command for that

Side note: use .github/workflows on Gitea if you don't use Gitea Actions Extension (yes there are multiple differences to allowed syntax)

> Gitlab, Radicle

I don't think this is a good idea, at least gitlab pipelines are fundermentally different
